components/PartnerBenefits.tsx

```diff
@@ -0,0 +1,42 @@
+import Link from "next/link";
+
+interface IButtonProps {
+  title: string
+  linkPath: string
+}
+
+export default function PartnerBenefits () {
+
+  function LinkButton({title, linkPath}: IButtonProps) {
+    return (
+      <Link
+        className="bg-cesium-orange py-4 px-6 rounded-lg text-xl font-bold text-white"
+        href={linkPath}
+      >
+        {title}
+      </Link>
+    )
+  }
+
+  return (
+      <div className="relative text-center mt-6 h-[400px] bg-light-gray">
+        <h1 className="pt-8 font-bold text-xl">
```

Comment:
`<h1 className="pt-8 font-bold text-2xl">`

It think if you put the text bigger, would be more similar of the design